config.ini.example

```diff
@@ -24,6 +24,7 @@ modules = simulation.modules.geometry.Geometry
           simulation.modules.save.FileOutput
           simulation.modules.macrophage.Macrophage
           simulation.modules.neutrophil.Neutrophil
+          simulation.modules.plot.Plot
```

Comment:
Can you update this branch by merging `origin/master` into `plot-module`? The recently-merged #77 may have conflicts.